packages/webapp/src/store/api/types/index.ts

```diff
@@ -0,0 +1,60 @@
+/*
+ *  Copyright 2024 LiteFarm.org
+ *  This file is part of LiteFarm.
+ *
+ *  LiteFarm is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation, either version 3 of the License, or
+ *  (at your option) any later version.
+ *
+ *  LiteFarm is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ *  GNU General Public License for more details, see <https://www.gnu.org/licenses/>.
+ */
+
+// If we don't necessarily want to type an endpoint
+export type Result = Array<{ [key: string]: any }>;
+
+export interface Animal {
```

Comment:
We need to add `internal_identifier` and `photo_url` to Animal and AnimalBatch ([PR](https://github.com/LiteFarmOrg/LiteFarm/pull/3125)), can it be done now??